openghg/store/base/_datasource.py

```diff
@@ -27,31 +30,55 @@ class Datasource:
     _datasource_root = "datasource"
     _data_root = "data"
```

Comment:
```suggestion
```
`_data_root` isn't used any more; this info has moved to `LocalZarrStore`

Reply:
Removed.